### PR TITLE
Wlpotter bl documentation 2024 06 14

### PR DESCRIPTION
**Pull request by wlpotter**
@davidamichelson here are the documentation changes I've made over the last week or so, mostly focusing on the sections assigned to me.

We can discuss any additional revisions. I haven't yet regenerated the HTML from the ODD; will do so once this gets approved and pulled in

### CODE REVIEW

File: documentation/schema/Syriacamsdesc.odd [hunk 1]
```diff
@@ -2014,11 +2135,75 @@
...
+                                    <cell>Syriac</cell><cell>syr</cell>
+                                </row>
+                                <row>
+                                    <cell>Syriac Garshuni in Arabic script</cell><cell>syr-Arab</cell>
+                                </row>
+                                <row>
+                                    <cell>Turkish</cell><cell>tr</cell>
+                                </row>
+                                <row>
+                                    <cell>Turkish Garshuni</cell><cell>tr-Syrc</cell>
```
Comment (davidamichelson) [inline documentation/schema/Syriacamsdesc.odd:2191]: @wlpotter Should we say Turkish Garshuni in Syriac Script ?

File: documentation/schema/Syriacamsdesc.odd [hunk 2]
```diff
@@ -2014,11 +2135,75 @@
...
+                               For this language, we use a code in the range (qaa-qtz) ISO 639 leaves 'reserved for local use'.
+                               While codes in this range are considered private, we follow the assignment published by
+                                <ref target="https://en.wikipedia.org/wiki/Wikipedia:WikiProject_Languages/List_of_ISO_639-3_language_codes_used_locally_by_Linguist_List">Linguist List</ref> for Middle Aramaic, that is qhy, which is further specified via a private use subtag resulting in the language tag: qhy-x-cpas.</p>
+                            <p>The accepted language codes are:</p>
+                            <table>
+                                <row>
+                                    <cell>Arabic</cell><cell>ar</cell>
+                                </row>
+                                <row>
+                                    <cell>Arabic Garshuni</cell><cell>ar-Syrc</cell>
```
Comment (davidamichelson) [inline documentation/schema/Syriacamsdesc.odd:2152]: @wlpotter Should we say Arabic Garshuni in Syriac Script

File: documentation/schema/Syriacamsdesc.odd [hunk 3]
```diff
@@ -4440,8 +4627,87 @@
...
+                        </list>
+                        <p>The <att>script</att> attribute provides a
+                            classification of the script. Oftentimes, only the broadest
+                            classification is possible. <title>SMBL</title> has used a combination
+                            of an ISO 639 Language Code with an optional <ref target="https://www.unicode.org/iso15924/">ISO 15924</ref> script code to encode a hand's script information. The <att>script</att> can take multiple, space-separated script values</p>
+                        <p>The accepted script values are:</p>
+                        <table>
+                            <row><cell>ar</cell><cell>Unspecified Arabic script</cell></row>
+                            <row><cell>ar-Syrc</cell><cell>Arabic Garshuni in Unspecified Syriac script</cell></row>
+                            <row><cell>ar-Syrn</cell><cell>Arabic Garshuni in East Syriac script</cell></row>
```
Comment (davidamichelson) [inline documentation/schema/Syriacamsdesc.odd:4660]: @wlpotter We do not have this in the list above. Should we add it?

File: documentation/schema/Syriacamsdesc.odd [hunk 4]
```diff
@@ -4634,8 +4900,251 @@
...
-                        <p>Further documentation will be added here.</p>
+                        <head>Decoration Description</head>
+                        <p><title>SMBL</title> has used <gi>decoNote</gi> elements within the <gi>decoDesc</gi>
+                           to record information about decorative features in a manuscript. Each <gi>decoNote</gi> element
+                           may have a <gi>locus</gi> element, which records the folio location(s) on which the decorative
+                           elements appear using <att>from</att>, and optionally <att>to</att>, attributes. A prose
+                           text node records Wright's description of the decorative feature, with the exception of <val>rubrication</val>
+                           where a text node was omitted if Wright does not provide additional details.</p>
+                        <p>Each <gi>decoNote</gi> element has a <att>type</att> attribute.
+                            <title>SMBL</title> uses a subset of the Fihrist vocabulary, with one addition <val>ornamentation</val> which
```
Comment (davidamichelson) [inline documentation/schema/Syriacamsdesc.odd:4911]: I see that you have properly used "val" here which is right, but the HTML transform does not render it, so I have been using "q". We could change mine or discuss.

File: documentation/schema/Syriacamsdesc.odd [hunk 5]
```diff
@@ -4634,8 +4900,251 @@
...
+                           elements appear using <att>from</att>, and optionally <att>to</att>, attributes. A prose
+                           text node records Wright's description of the decorative feature, with the exception of <val>rubrication</val>
+                           where a text node was omitted if Wright does not provide additional details.</p>
+                        <p>Each <gi>decoNote</gi> element has a <att>type</att> attribute.
+                            <title>SMBL</title> uses a subset of the Fihrist vocabulary, with one addition <val>ornamentation</val> which 
+                           refers to any sort of undifferentiated ornamentation. The allowed values are as follows:</p>
+                        <list>
+                            <item>border</item>
+                            <item>diagram</item>
+                            <item>frieze</item> <!-- no examples in SMBL -->
```
Comment (davidamichelson) [inline documentation/schema/Syriacamsdesc.odd:4916]: Lets discuss these comments and delete the unused ones I think?

File: documentation/schema/Syriacamsdesc.odd [hunk 6]
```diff
@@ -4634,8 +4900,251 @@
...
+                                    <decoNote type="illustration">
+                                        The figure of a fish on the lower margin of <locus from="14b" to="14b">fol. 14b</locus>.
+                                    </decoNote>
+                                </decoDesc>
+                            </egXML>
+                        </p>
+                        <p>Optionally, <title>SMBL</title> has employed a <att>subtype</att> attribute to incorporate
+                            the decoration vocabulary of the <ref target="https://www.vhmml.org/">Hill Museum and Manuscripts Library</ref>.</p>
+                        <p>The following terms derive from the <ref 
+                            target="https://docs.google.com/spreadsheets/d/1FZbPVbFKq_omXklMbJwxC9_85aarcJYFkt5A_FLUayU/edit?gid=1769902322#gid=1769902322">vHMML Features List</ref>:</p>
```
Comment (davidamichelson) [inline documentation/schema/Syriacamsdesc.odd:4951]: Do we also want to link to HMML itself?

**Review by davidamichelson — APPROVED** [review]
@wlpotter Here are my comments, great work!

File: documentation/schema/Syriacamsdesc.odd (hunk 3 above)
Comment (wlpotter) [reply documentation/schema/Syriacamsdesc.odd:4660]: The difference here is that these are specific `@script` values, not all of which should be used for `@xml:lang` or `@mainLang` attributes

File: documentation/schema/Syriacamsdesc.odd (hunk 6 above)
Comment (wlpotter) [reply documentation/schema/Syriacamsdesc.odd:4951]: Yes, that's probably a good idea

File: documentation/schema/Syriacamsdesc.odd (hunk 5 above)
Comment (wlpotter) [reply documentation/schema/Syriacamsdesc.odd:4916]: Sounds good, I couldn't tell if you'd left them in other places, so didn't want to delete without confirming

File: documentation/schema/Syriacamsdesc.odd (hunk 1 above)
Comment (davidamichelson) [reply documentation/schema/Syriacamsdesc.odd:2191]: So Syro-Turkish

File: documentation/schema/Syriacamsdesc.odd (hunk 3 above)
Comment (davidamichelson) [reply documentation/schema/Syriacamsdesc.odd:4660]: Syro-Arabic Garshunography in East Syriac script

Comment (davidamichelson) [reply documentation/schema/Syriacamsdesc.odd:4660]: Syro-Arabic Garshunography in unspecified Syriac script

File: documentation/schema/Syriacamsdesc.odd [hunk 7]
```diff
@@ -4440,8 +4627,87 @@
...
+                            <row><cell>en</cell><cell>Unspecified English script</cell></row>
+                            <row><cell>fr</cell><cell>Unspecified French script</cell></row>
+                            <row><cell>grc</cell><cell>Unspecified Greek script</cell></row>
+                            <row><cell>he</cell><cell>Unspecified Hebrew script</cell></row>
+                            <row><cell>hy</cell><cell>Unspecified Armenian script</cell></row>
+                            <row><cell>it</cell><cell>Unspecified Italian script</cell></row>
+                            <row><cell>la</cell><cell>Unspecified Latin script</cell></row>
+                            <row><cell>qhy-x-cpas</cell><cell>Christian Palestinian Aramaic script</cell></row>
+                            <row><cell>syr</cell><cell>Unspecified Syriac script</cell></row>
+                            <row><cell>syr-Arab</cell><cell>Syriac Garshuni in Unspecified Arabic script</cell></row>
```
Comment (davidamichelson) [inline documentation/schema/Syriacamsdesc.odd:4672]: Arabo-Syriac garshunography in unspecified Arabic script

File: documentation/schema/Syriacamsdesc.odd [hunk 8]
```diff
@@ -4440,8 +4627,87 @@
...
+                            <row><cell>syr-Syre</cell><cell>Estrangela script</cell></row>
+                            <row><cell>syr-Syrj</cell><cell>West Syriac script</cell></row>
+                            <row><cell>syr-Syrn</cell><cell>East Syriac script</cell></row>
+                            <row><cell>syr-x-syrm</cell><cell>Melkite Syriac script</cell></row>
+                            <row><cell>tr</cell><cell>Unspecified Turkish script</cell></row>
+                            <row><cell>tr-Syrc</cell><cell>Turkish Garshuni in Unspecified Syriac script</cell></row>
```
Comment (davidamichelson) [inline documentation/schema/Syriacamsdesc.odd:4678]: Syro-Ottoman Turkish garshunography in unspecified Syriac script

File: documentation/schema/Syriacamsdesc.odd (hunk 5 above)
Comment (davidamichelson) [reply documentation/schema/Syriacamsdesc.odd:4916]: Move unused ones down to the diverge from consolidated section.

**Pull request merged**